src/index.css

```diff
@@ -1,8 +1,21 @@
+@import url('https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,400;1,300;1,400&display=swap');
+
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
 
-body {
-  font-family: CreativeVintageDraft;
-  /* font-family: Montserrat-Regular; */
-}
\ No newline at end of file
+@font-face {
+  font-family: 'PilcrowRounded-Variable';
+  src: url('../fonts/PilcrowRounded-Variable.woff2') format('woff2'),
+       url('../fonts/PilcrowRounded-Variable.woff') format('woff'),
+       url('../fonts/PilcrowRounded-Variable.ttf') format('truetype');
+       font-weight: 400 900;
```

Comment:
are you sure this will work? I think font-weight can only have one value at a time